cmd/src/setup.rs

```diff
@@ -95,18 +101,39 @@ pub fn initial_logger(config: &TiKvConfig) {
                     e
                 );
             });
-            let drainer = logger::LogDispatcher::new(drainer, slow_log_drainer);
-            logger::init_log(
-                drainer,
-                config.log_level,
-                true,
-                true,
-                vec![],
-                config.slow_log_threshold.as_millis(),
-            )
-            .unwrap_or_else(|e| {
-                fatal!("failed to initialize log: {}", e);
-            });
+
+            match config.log_format {
+                LogFormat::Text => build_loggers(
+                    logger::text_format(writer),
+                    logger::text_format(slow_log_writer),
+                    config,
+                ),
+                LogFormat::Json => build_loggers(
+                    logger::json_format(writer),
+                    logger::json_format(slow_log_writer),
+                    config,
+                ),
+            };
+
+            fn build_loggers<N, S>(normal: N, slow: S, config: &TiKvConfig)
```

Comment:
The name is similar with `build_logger`, better give it a different name or just use the same function.

Reply:
fixed at e843588